asciidtype/asciidtype/src/casts.c

```diff
@@ -102,3 +287,59 @@ PyArrayMethod_Spec ASCIIToASCIICastSpec = {
         .dtypes = a2a_dtypes,
         .slots = a2a_slots,
 };
+
+static PyType_Slot u2a_slots[] = {
+        {NPY_METH_resolve_descriptors, &unicode_to_ascii_resolve_descriptors},
+        {_NPY_METH_get_loop, &unicode_to_ascii_get_loop},
+        {0, NULL}};
+
+static char *u2a_name = "cast_Unicode_to_ASCIIDType";
+
+static PyType_Slot a2u_slots[] = {
+        {NPY_METH_resolve_descriptors, &ascii_to_unicode_resolve_descriptors},
+        {_NPY_METH_get_loop, &ascii_to_unicode_get_loop},
```

Comment:
No need for get_loop, can just use `NPY_METH_strided_loop` directly.  OTOH, maybe it doesn't add a lot of convenience anyways...